src/objects/featured-row/featured-row.stories.mdx

```diff
@@ -0,0 +1,18 @@
+import { Story, Canvas, Meta } from '@storybook/addon-docs/blocks';
+import defaultDemo from './demo/demo.twig';
+
+<Meta title="Objects/Featured Row" />
+
+# Featured Row
```

Comment:
Nit: Wondering if "Feature Row" works just as well as the past-tense "Featured"?

Also wondering if "Row" is any more accurate than "Group" or something like that?